test/configuration_test.dart

```diff
@@ -638,3 +636,9 @@ Future<void> main([List<String>? args]) async {
     expect(() => realm.write(() {}), throws<RealmException>("Number of active versions (3) in the Realm exceeded the limit of 2"));
   });
 }
+
+Future<void> _copyFile(String fromPath, String toPath) async {
+  await File(fromPath).copy(toPath);
+}
+
+var copyFile = _copyFile; // default, but allow integration_test to override
```

Comment:
Do we want this to live in `configuration_test.dart` or more generally in `test.dart`? I feel like that's where we put helper functions like this one.

Reply:
I'll move it. Currently it is only used by `configuration_test.dart`